nix-libs/nodeLib/buildNodePackage.nix

```diff
@@ -329,7 +329,10 @@ let
                    "'package' directory. Don't know how to handle this :("
               exit 1
             fi
-            mv package node_modules/${dep.fullName}
+
+            mkdir -p node_modules/${dep.fullName}
+            cp -rP package/. node_modules/${dep.fullName}
```

Comment:
we could still use `mv` if we just call `mkdir -p $(dirname node_modules/${dep.fullName})` first

But I suppose this would only make a difference if the dependency was large

Reply:
The reason why I chose `cp` over `mv` was for clarity.

I often find myself confused by whether I have moved a folder _underneath_ the target directory or if I _renamed_ it to the target directory and I think that becomes even more confusing with the interpolated name which may represent a directory tree and not just a single-level directory.

I agree that the size of the package might matter when copying vs. renaming but I don't think we need to worry _too_ much about that.